lib/login.dart

```diff
@@ -191,7 +191,6 @@ class BronzeRequestApiKeyFormState extends State<BronzeRequestApiKeyForm> {
         ),
         body: BiforgePage(
             scrollChild: true,
-            showDebugInfo: true,
```

Comment:
dont remove this from the login screens, just the main screen where we have the sidebar option